e2etest/test_manager.go

```diff
@@ -205,7 +205,9 @@ func (tm *TestManager) RetrieveTransactionFromMempool(t *testing.T, hashes []*ch
 	var txs []*btcutil.Tx
 	for _, txHash := range hashes {
 		tx, err := tm.BTCClient.GetRawTransaction(txHash)
-		require.NoError(t, err)
+		if err != nil {
+			continue
+		}
```

Comment:
wdyt we return error upon this? it's possible that some hash is missing and it's the caller's responsibility to retry